### DISCUSSION TITLE
Making changes to display

### DISCUSSION THREAD

**Discussion by Arrgg** [Blackymas/NSPanel_HA_Blueprint#2313] [Q&A]
I feel like an idiot here, but do you have to start from scratch every time you want to make a change to a panel via gui?  Said another way, can a configuration be re-loaded into the blueprint for further editing?

**Comment by edwardtfn** [comment]
Once you have created an automation using the Blueprint you can freely edit that automation and all the new settings will be sent to the panel when you save it.
There's no need to create a new automation every time.
